packages/kbn-securitysolution-exceptions-common/api/model/exception_list_common.gen.ts

```diff
@@ -60,23 +57,24 @@ in which it is created, where:
 
   */
 export type ExceptionNamespaceType = z.infer<typeof ExceptionNamespaceType>;
-export const ExceptionNamespaceType = z.enum(['agnostic', 'single']);
-export type ExceptionNamespaceTypeEnum = typeof ExceptionNamespaceType.enum;
-export const ExceptionNamespaceTypeEnum = ExceptionNamespaceType.enum;
+export const ExceptionNamespaceType = z.enum(['agnostic', 'single']).default('single');
```

Comment:
We should not include default values in schema primitives. Default values are context-dependent, and a schema might have different defaults in different contexts or might not have a default value at all. For example, we deliberately excluded all default values from rule schemas because different endpoints, like rule update or rule patch, treat empty values very differently. With a default value incorporated into the schema, performing certain actions requiring a missing value becomes impossible. Once the payload is parsed, we cannot distinguish if the value was missing in the request and substituted by the schema itself or if the payload contained a value. We need to reconsider this approach. Let's sync on the alternatives we have.

Reply:
@maximpn What happens if we disable the `no-$ref-siblings` rule and continue shipping OpenAPI schemas that are not fully compliant with 3.0?

Additionally, what effort is required to upgrade to 3.1? Are there any blockers?